Server/App/Infrastructure/Mapping/Endpoints/Concrete/AuthEndpoints.cs

```diff
@@ -68,28 +80,69 @@ public void MapRoutes(IEndpointRouteBuilder routeBuilder)
             {
                 return Results.BadRequest("Wrong password");
             }
-            
+
             var claims = new[]
             {
                 new Claim(ClaimTypes.Name, user.Name), //TODO: claims should be extended
                 new Claim(ClaimTypes.Role, user.Role),
+                new Claim(ClaimTypes.Email, user.Email)
             };
 
-            var token = new JwtSecurityToken
-            (
+            var token = new JwtSecurityToken(
                 issuer: configuration["Jwt:Issuer"],
                 audience: configuration["Jwt:Audience"],
                 claims: claims,
-                expires: DateTime.UtcNow.AddMinutes(15), //TODO: should be changed
-                notBefore: DateTime.UtcNow,
+                expires: DateTime.UtcNow.AddMinutes(15),
                 signingCredentials: new SigningCredentials(
                     new SymmetricSecurityKey(Encoding.UTF8.GetBytes(configuration["Jwt:Key"])),
                     SecurityAlgorithms.HmacSha256)
             );
-            
             var tokenString = new JwtSecurityTokenHandler().WriteToken(token);
-            
+
+            // Сохранение куки для последующего входа
```

Comment:
Cookie expire date is now set to "session", so after closing browser/application it`s expires. Is it have to work like that or expire date have to be longer?